letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
I agree that's weird. That code just landed from #1969, hasn't shipped yet, and looks buggy to me. @bmw @eldering either of you know what was going on here?